matrix/plugins/glitch/actions.py

```diff
@@ -19,21 +22,23 @@ class _Actions(dict, metaclass=Singleton):
     functions.
 
     """
-    def decorate(self, func):
+    def decorate(self, object_type):
```

Comment:
Whoops. That's remaining from when I was passing in object type explicitly, with the double nested decorators. Should be "func"